docs/current/742989-cookbook.md

```diff
@@ -625,6 +625,44 @@ Set the Hashicorp Vault URI, namespace, role and secret identifiers as host envi
 
 [Learn more](./guides/723462-use-secrets.md)
 
+## Error handling
+
+### Aborting with unhandled error
+
+The following code listing demonstrates how to abort the script with an unhandled error, which allows debugging with the stack trace of the crash during development.
+
+```go file=./cookbook/snippets/error-handling/unhandled/main.go
+```
+
+### Aborting gracefully
+
+The following code listing demonstrates how to abort the script without crashing.
+
+```go file=./cookbook/snippets/error-handling/aborting/main.go
+```
+
+### Exit code and unexpected errors
+
+The following code listing demonstrates how to handle a non-zero exit code from running a command, with several use cases:
```

Comment:
```suggestion
The following code listing demonstrates how to handle a non-zero exit code or an error from running a command in a container, with several use cases:
```

Reply:
"non-zero exit code or an error from running a command in a container" reads ambiguously as either two possibilities or a clarification of same thing using different words. The intent is the latter, if you think that's clear than I'm 👍

Reply:
See if clearer now?